aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -22,7 +22,7 @@ namespace {
 template <class T>
 struct Vec256 {
   static constexpr int size = 32 / sizeof(T);
-  __at_align32__ T values[32 / sizeof(T)];
+  T values[32 / sizeof(T)];
```

Comment:
Was the alignment unnecessary here?

Reply:
It causes that nasty compiler warning and is unlikely to help.

Reply:
@cpuhrsch Do you think there would be any harm to add this back? It's probably useful in speeding up some bitwise operations.

Reply:
Deferring the answer to @VitalyFedyunin